packages/node_modules/nav-frontend-skjema/src/checkboks-panel-gruppe.tsx

```diff
@@ -24,21 +24,27 @@ export interface CheckboksPanelGruppeProps {
      * Objekt som beskriver skjema-feil, se 'skjemaelement-feilmelding.tsx'
      */
     feil?: SkjemaelementFeil;
+    /**
+     * Styling klasse for container
+     */
+    className?: string;
 }
 
 class CheckboksPanelGruppe extends React.Component<CheckboksPanelGruppeProps> {
     render() {
-        const { checkboxes, legend, feil, onChange } = this.props;
+        const { checkboxes, legend, feil, onChange, className } = this.props;
```

Comment:
Endrer du til `const { checkboxes, legend, feil, onChange, className, ...other } = this.props;` her?